doc/configuration/tokens/yubikey.rst

```diff
@@ -30,4 +25,89 @@ Run the command like this::
    privacyidea -U https://your.privacyidea.server -a admin token \
    yubikey_mass_enroll --yubimode YUBICO
 
+This command initializes the token and stores the AES secret and prefix
+in privacyidea, so the toen is immediatly useful. You can choose the slot
+with ``--yubislot`` For further help call
```

Comment:
I think here is a full stop missing:

with ""--yubislot"". For further help call